charts/immich/values.yaml

```
@@ -26,6 +26,35 @@ immich:
       # Automatically creating the library volume is not supported by this chart
       # You have to specify an existing PVC to use
       existingClaim:
+  # configuration is immich-config.json converted to yaml
+  # ref: https://immich.app/docs/install/config-file/
+  #
+  # WARNING: for storageTemplate->template it contains `{{` and `}}` which interferes with templating
+  # replace All : {{ with {{`{{`}}  AND  }} with {{`{{`}}
+  # for more examples see PR https://github.com/immich-app/immich-charts/pull/76
+  # 
+  # If not provided or empty, configuration can be changed from UI
+  #
+  # {
+  #   "trash": {
+  #     "enabled": true,
+  #     "days": 30
+  #   },
+  #   "storageTemplate": {
+  #     "enabled": true
+  #     "template": "{{y}}/{{y}}-{{MM}}-{{dd}}/{{filename}}"
+  #   }
+  # }
+  # can be provided as example below.
+  #
+  configuration: {}
+    # Example 
+    # trash:
```

```suggestion
  configuration: {}
    # trash:
```
It being commented out already indicates that it's an example ;)

---

okay :+1: